app/concerns/wpcc/salary_threshold.rb

```diff
@@ -0,0 +1,9 @@
+module Wpcc
+  module SalaryThreshold
+    extend ActiveSupport::Concern
+
+    def load_config(file:)
+      YAML.load_file(Wpcc::Engine.root.join('config', "#{file}.yml"))
+    end
+  end
+end
```

Comment:
@tomas-stefano 
This class will remain as the other concern class which slipped in has been removed